src/browser/screens/Fourth.react.js

```diff
@@ -101,15 +85,18 @@ export default class FourthScreen extends Component {
           </div>
           <ul>
             {skillsForCurrentRole.map((skill) =>
-              <SkillItem actions={this.props.actions} data={skill} key={skill.id} resetErrorStatus={this.resetErrorStatus.bind(this)}/>
+              <SkillItem actions={this.props.actions} data={skill} key={skill.id}/>
             )}
             <li className="otherSkill">
-              {hint}
-              <div className="otherSkill-img">
-                <img {...cb} alt="Other skills" src="../../../assets/img/skills/skills_gears.svg"/>
+              {this.state.hintDisplayed ? <div className='hint'>Other skills</div> : false};
```

Comment:
Maybe it would be more readable if you would declared those two as a variables. IMO this ternary is not that readeable